pkg/sanitization/html_test.go

```diff
@@ -0,0 +1,72 @@
+package sanitization
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type testSanitizeHTMLType struct {
+	testCase       string
+	input          string
+	expectedOutput string
+}
+
+func TestDevelopmentHTML(t *testing.T) {
+	testCase := testSanitizeHTMLType{
+		testCase:       "Development",
+		input:          `<a href="https://dev.easi.cms.gov/">Valid link</a>`,
+		expectedOutput: `<a href="https://dev.easi.cms.gov/" rel="nofollow noreferrer noopener" target="_blank">Valid link</a>`,
+	}
+	actualOutput := SanitizeHTML(testCase.input)
+	assert.EqualValues(t, testCase.expectedOutput, actualOutput)
+
+}
+func TestSanitizeHTML(t *testing.T) {
+
+	sanitizeHTMLTests := []testSanitizeHTMLType{
```

Comment:
Can you add a quick test case for disallowing relative URLs in `<a>` tags?

Reply:
Since the FE automatically adds `https://` to relative links ([source](https://github.com/CMSgov/easi-app/blob/9bac915dadde8de36dc19de8e22a2c00eb9381c0/src/components/RichTextEditor/index.tsx#L214-L226)) I think it makes sense to disallow them from the BE, since we don't expect users to be sending them

Reply:
Sounds good! I'll update the policy

Reply:
Thanks Clay, I've updated the policy now, and updated the test suites to demonstrate that relative links are removed.